net/nimble/controller/src/ble_ll_adv.c

```diff
@@ -1310,20 +1308,40 @@ ble_ll_set_adv_secondary_start_time(struct ble_ll_adv_sm *advsm)
 {
     static const uint8_t bits[8] = {0, 1, 1, 2, 1, 2, 2, 3};
     struct ble_ll_sched_item *sched = &advsm->adv_sch;
-    uint32_t ext_duration;
+    uint32_t adv_pdu_dur;
+    uint32_t adv_event_dur;
     uint8_t chans;
 
     assert(advsm->adv_chanmask <= BLE_HCI_ADV_CHANMASK_DEF);
```

Comment:
(I know this is not related to that change but..:P) this assert should be ( mask <= DEF && mask > 0)